subprojects/core/src/integTest/groovy/org/gradle/api/tasks/ArchiveIntegrationTest.groovy

```diff
@@ -276,7 +273,13 @@ class ArchiveIntegrationTest extends AbstractIntegrationSpec {
                 from tarTree('compressedTarWithWrongExtension.tar')
                 into 'dest'
             }
-'''
+        '''.stripIndent()
+        // We cannot use `TestResources` since we change the method name with `TestReproducibleArchives`
```

Comment:
I found this commit comment: 
"Creating an improper tar every time the test runs proved unsafe as sometimes the produced archive was good enough for the untarring to run"
https://github.com/gradle/gradle/commit/43c7d2997a8411ef458067a7bfa2eef14e9fec01